docs/scanners/gitleaks.md

```diff
@@ -0,0 +1,43 @@
+# [Gitleaks](https://github.com/zricethezav/gitleaks)
+
+The [Gitleaks Scanner](https://github.com/zricethezav/gitleaks) tool for finding secrets within a directory or Git repo.
+
+## Configuration
+
+Gitleaks currently always runs with the `--recursive` and `--format json` options turned on.
+In addition, you can configure the additional options below.
+For more information on Gitleaks configuration, see the docs [here](https://github.com/zricethezav/gitleaks#usage-and-options)
+
+```yaml
+  scanner_configs:
+    Gitleaks:
```

Comment:
Can you write a brief comment next to each of these config options. For example,
```yaml
config-path: "./gitleaks.conf"   # Path to local gitleaks config
...
```